### PR TITLE
License Page

### PR DESCRIPTION
**Pull request by EllDoubleYew**
#### This pull request addresses:

Addresses https://github.com/unfoldingWord-dev/translationCore/issues/967

#### How to test this pull request:

Click Home in the top left, click on the licenses tab, the information should display there.

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1215)

### CODE REVIEW

**Comment by mannycolon** [comment]
Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[src/js/components/core/licenses/Licenses.js, line 8 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1215#-KibfVb5zw0t1PrDZkhV:-KibfVb5zw0t1PrDZkhW:ba7dlyd) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/d3d872d19c03fa9239b2b94bac377086634e18ad/src/js/components/core/licenses/Licenses.js#L8)):*
> ```JavaScript
>     return(
>       <div>
>         <center>{"Find out more about translationCore at: "}<a href="http://unfoldingword.org/tc">{"unfoldingword.org/tc"}</a></center>
> ```

you only need one center tag for all of this elements

in other words you dont need a center tag for each element.

---

*[src/js/components/core/licenses/ThirdParty.js, line 8 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1215#-KibfqgTpQ9W_e9v2-5I:-KibfqgTpQ9W_e9v2-5J:b8g46e4) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/d3d872d19c03fa9239b2b94bac377086634e18ad/src/js/components/core/licenses/ThirdParty.js#L8)):*
> ```JavaScript
>     let libraries = []
>     for(let license in Licenses){
>       console.log(license);
> ```

do we still need the console log??

---

*[src/js/components/core/licenses/ThirdParty.js, line 11 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1215#-KibftINnv-YAuL2UlXX:-KibftINnv-YAuL2UlXY:bb7hzsp) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/d3d872d19c03fa9239b2b94bac377086634e18ad/src/js/components/core/licenses/ThirdParty.js#L11)):*
> ```JavaScript
>       libraries.push(
>         <div style={{padding: "20px"}}>
>           <center>{license}</center>
> ```

ibid

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1215)*

**Comment by klappy** [comment]
*[src/js/components/core/licenses/Licenses.js, line 8 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1215#-KibfVb5zw0t1PrDZkhV:-KijzShjRsGmtJ8ADWTX:bcddj6) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/d3d872d19c03fa9239b2b94bac377086634e18ad/src/js/components/core/licenses/Licenses.js#L8)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

you only need one center tag for all of this elements

in other words you dont need a center tag for each element.
</blockquote></details>

@elldoubleyew: Can you address this with other tags and only use one center? or even make the parent div have css text-align: center... https://css-tricks.com/almanac/properties/t/text-align/

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/1215)*

**Comment by EllDoubleYew** [comment]
@klappy @mannycolon The idea here was semantic markup. I wanted to see if I could do this CSS-Free. I am well aware that it can all sit in the same div and be centered via CSS 😉. Just let me know which way you want to go.

**Comment by klappy** [comment]
Can you also update with develop and resolve conflicts?
